docs/manual/manual_stock_led.rst

```diff
@@ -0,0 +1,251 @@
+
+    # Using a Vertiq 40-06 Gen 2 module with speed firmware
+    module = iq.Vertiq4006(com, firmware="speed")
+
+    #Turn on the white LED to minimum brightness
+    module.set("white_led", "intensity", 1)
+
+If you restart your module now, you will notice that the white LED does not come on. This is because the white LED ``intensity`` parameter was not saved. On boot, 
+the LED's intensity is configured to the value read from the module's persistent memory. To save a desired intensity, you can either use a ``save`` 
+:ref:`command <api_interactions>`, or you can use IQ Control Center. After connecting your module to the Control Center, you will find the following White 
+LED parameters on the General tab
+
+.. image:: ../_static/manual_images/leds/white_led_cc_params.png
+
+Suppose you would like to drive the LED with 2% intensity on boot, simply set ``White LED Intensity`` to 2. Now, after rebooting your module, you will see that the 
+white LEDs now turn on with the same intensity as set and saved previously.
+
+.. _strobing_configuration:
+
+Strobing Configuration
+========================
+Strobing Enable
+******************
+The ``strobe_enable`` parameter determines whether or not strobing is applied. If ``strobe_enable`` is false (0), the corresponding LED remains static according 
+to its intensity values. For example, suppose we want the white LED to be on at all times at a low intensity. To do so, we use IQ Control Center to 
+set ``White LED Strobing Active`` to ``Inactive``, and ``White LED Intensity`` to 1.
+
+.. image:: ../_static/manual_images/leds/white_strobe_off.png
+
+For the sake of visibility in this example, set ``White LED Intensity`` to 0 to turn off the white LED. Now, suppose we want the RGB LED to strobe red only. 
+Still in the Control Center, find the RGB LED parameters, and set ``RGB LED Blue`` and ``RGB LED Green`` to 0, ``RGB LED Red`` to 150, and 
+set ``RGB LED Update Color``. Assuming that RGB strobing is disabled (which is the default configuration), your LED should now be statically red.
+
+.. image:: ../_static/manual_images/leds/red_only_on.png
+
+The RGB LED is configured with the same default pattern as the white LED, 3 blinks on and off followed by a longer period off, and the pattern repeats. 
+So, when we set ``RGB LED Strobing Active`` to ``Active``, we will see the red LED start to blink.
+
+Strobing activation can be changed on the fly, and is configured on boot-up according to the value stored in the module's persistent memory.
+
+<GET A VIDEO OF THIS (MAYBE ANOTHER SCREEN CAP)>
+
+.. _strobing_period:
+
+Strobing Period
+******************
+The strobing period defines the time it takes the module to complete one full cycle through the strobing pattern. This parameter only has an effect when strobing is 
+active. Strobe period is defined in seconds. Let's take the default strobing pattern used for both the RGB and white LEDs (blink, blink, blink, off). 
```

Comment:
The (blink, blink, blink, off) is not the most precise way to convey this pattern. Maybe a diagram like the one you have below, maybe a video, maybe just a more elaborate description, but this description is a bit underspecified at the moment.